pom.xml

```diff
@@ -38,6 +38,8 @@
         <commons-lang.version>2.6</commons-lang.version>
         <spotless.version>2.4.2</spotless.version>
         <jaxb-api.version>2.3.1</jaxb-api.version>
+        <!-- rocketmq-schema-registry need to compile by yourself for the time being -->
```

Comment:
Removes the comments for rocketmq-schema-registry maven dependencies. The rocketmq-schema-registry is deploying to the maven repository.

Reply:
> Removes the comments for rocketmq-schema-registry maven dependencies. The rocketmq-schema-registry is deploying to the maven repository.

Since the release version of 1.0.0 has been merged,I will remove it.